encoding.h

```diff
@@ -68,6 +68,10 @@
 #define MCONTROL_STORE      (1<<1)
 #define MCONTROL_LOAD       (1<<0)
 
+#define MCONTROL_MASKMAX(xlen) (0x3f<<((xlen)-11))
```

Comment:
I wasn't sure what the convention for those was. This is a good solution. I'll change the spike code to use them.